src/components/displaySettings/displaySettings.template.html

```diff
@@ -6,66 +6,75 @@ <h2 class="sectionTitle">
     <div class="selectContainer languageSection hide">
         <select id="selectLanguage" is="emby-select" label="${LabelDisplayLanguage}">
             <option value="">${Auto}</option>
-            <option value="af">Afrikaans</option>
-            <option value="sq">Albanian</option>
-            <option value="ar">Arabic</option>
-            <option value="be-BY">Belarusian</option>
-            <option value="bn_BD">Bengali (Bangladesh)</option>
-            <option value="bg-BG">Bulgarian</option>
-            <option value="ca">Catalan</option>
-            <option value="zh-HK">Chinese (Hong Kong)</option>
-            <option value="zh-CN">Chinese (Simplified)</option>
-            <option value="zh-TW">Chinese (Traditional)</option>
-            <option value="hr">Croatian</option>
-            <option value="cs">Czech</option>
-            <option value="da">Danish</option>
-            <option value="nl">Dutch</option>
+            <option value="af">Afrikaans (af)</option>
+            <option value="ar">العربية (ar)</option>
+            <option value="be-BY">Беларуская (be)</option>
+            <option value="bg-BG">Български (bg)</option>
+            <option value="bn_BD">বাংলা (bn-BD)</option>
+            <option value="ca">Català (ca)</option>
+            <option value="cs">Čeština (cs)</option>
+            <option value="cy">Cymraeg (cy)</option>
+            <option value="da">Dansk (da)</option>
+            <option value="de">Deutsch (de)</option>
+            <option value="el">Ελληνικά (el)</option>
+            <option value="en-GB">English (en-GB)</option>
             <option value="en-US">English</option>
-            <option value="en-GB">English (United Kingdom)</option>
-            <option value="eo">Esperanto</option>
-            <option value="fil">Filipino</option>
-            <option value="fi">Finnish</option>
-            <option value="fr">French</option>
-            <option value="fr-CA">French (Canada)</option>
-            <option value="gl">Galician</option>
-            <option value="de">German</option>
-            <option value="gsw">German (Swiss)</option>
-            <option value="el">Greek</option>
-            <option value="he">Hebrew</option>
-            <option value="hi-IN">Hindi</option>
-            <option value="hu">Hungarian</option>
-            <option value="is">Icelandic</option>
-            <option value="id">Indonesian</option>
-            <option value="it">Italian</option>
-            <option value="ja">Japanese</option>
-            <option value="kk">Kazakh</option>
-            <option value="ko">Korean</option>
-            <option value="lt-LT">Lithuanian</option>
-            <option value="ms">Malay</option>
-            <option value="mr">Marathi</option>
-            <option value="nb">Norwegian Bokmål</option>
-            <option value="fa">Persian</option>
-            <option value="pr">Pirate</option>
```

Comment:
I think it was added for fun, and it is still on Weblate.

Reply:
Hi
You know that you can add any language from the global list to Weblate, for example, Church Slavonic. But will it be practical and should we increase the amount of strings that will not be explicitly used?

Reply:
@WWWesten Please return `Pirate`. Team wants to keep it (at least for now) just for fun :smile: